less/sidemenu.less

```diff
@@ -52,7 +52,7 @@ body.menu-toggled #content-toggled {
   background-color: white;
   margin: 0;
   white-space: nowrap;
-  font-size: 1em;
+  font-size: 14px; /* force font size */
```

Comment:
with the modifications already applied, this should'nt be needed. Keep 1em.

Reply:
i've made it on purpose, to be sure that the menu item do exactly 14px. I don't want that this font-size depend of any other CSS parameter. Otherwise, any modification may break the display of advertising.

Reply:
Neverless, if we use 1rem, the base font-size won't change, so there is no fear for this. Moreover, if this is a concern, we could handle the leine hieght another way.
I suggest we use 1rem.